packages/webpack/__e2e-fixtures__/typescript/styles.css

```diff
@@ -0,0 +1,5 @@
+$MARGIN: 40px;
```

Comment:
Do these styles make sense at all? There is no SASS or similar loader defined to resolve the variable.

Suggestion: Drop the stylesheet completely 😉